apps/settings/js/panels/usb_storage/usb_transfer.js

```diff
@@ -217,6 +209,51 @@ define(function(require) {
     },
 
     /**
+     * update ums description and protocol selection fields
+     */
+    _updateUmsState: function ut_updateUmsState() {
+      var key;
+      Debug('enabled:' + this._elements.usbEnabledCheckBox.checked + '/' +
+            'volumeState:' + MediaStorage.volumeState);
+      if (this._elements.usbEnabledCheckBox.checked) {
+        key = 'enabled';
+        if (MediaStorage.volumeState === 'shared') {
+          this._disableProtocolSelections(true);
+        } else {
+          this._disableProtocolSelections(false);
+        }
+      } else if (MediaStorage.volumeState === 'shared') {
+        key = 'umsUnplugToDisable';
+        this._disableProtocolSelections(true);
+      } else {
+        key = 'disabled';
+        this._disableProtocolSelections(false);
+      }
+      this._elements.usbEnabledInfoBlock.setAttribute('data-l10n-id', key);
+    },
+
+    /**
+     * [_disableProtocolSelections description]
+     * @type {boolean} enabled to disable Protocol Selections
+     */
+    _disableProtocolSelections:
+      function ut_disableProtocolSelections(enabled) {
```

Comment:
Nit: can we rename this internal argument to 'disable' or something similar because it's a bit confusing in context of the function's name.